apis/fluent-en_GB/extensions/kotlin_1_3/atrium-api-fluent-en_GB-kotlin_1_3-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/kotlin_1_3/ResultFeatureAssertionsSpec.kt

```diff
@@ -34,6 +38,9 @@ class ResultFeatureAssertionsSpec : ResultFeatureAssertionsSpec(
 
             a3.isSuccess()
             a3 = a3.isSuccess { }
+
+            a4.isFailure<IllegalArgumentException>()
```

Comment:
please move these two lines up to line 33 and use a1 and a2 instead of a4

Reply:
By, 'these two lines' do you mean lines 41 (the empty line) and 42

Reply:
42 and 43, seems like I only marked 42

Reply:
Is this what you mean
```
a1.isSuccess()
a2.isSuccess()
a1 = a1.isSuccess { }
a2 = a2.isSuccess { }
a1.isFailure<IllegalArgumentException>()
a2.isFailure<IllegalArgumentException>()
            
val r1: Expect<IllegalArgumentException> = a1.isFailure<IllegalArgumentException> {  }
val r2: Expect<IllegalArgumentException> = a2.isFailure<IllegalArgumentException> {  }
```

Reply:
almost, just add the following before `val r1...`
```
a1.isFailure<IllegalArgumentException>()
a2.isFailure<IllegalArgumentException>()
```

Reply:
sorry, it is there :man_facepalming: please add the blank line between isSucecss and isFailure, this way people like me are not confused :laughing: